admin/resources/js/components/PoolDashboard.tsx

```diff
@@ -197,12 +194,26 @@ export const PoolDashboard: React.FC = () => {
       href="/operational-requirements"
       text={intl.formatMessage(messages.menuOperationalRequirements)}
     />,
-    <MenuLink
-      key="pool-candidates"
-      href="/pool-candidates"
-      text={intl.formatMessage(messages.menuPoolCandidates)}
-    />,
   ];
+
+  if (!fetching && !error) {
+    menuItems.push(
+      <MenuHeading
+        key="pool-candidates"
+        text={intl.formatMessage(messages.menuPoolCandidates)}
+      />,
```

Comment:
Shouldn't the menu have a menu heading reading "Pools" with button-links underneath to each pool. Then when the user clicks a pool it shows a list of the pool candidates in that pool?  
![image](https://user-images.githubusercontent.com/22059495/129778794-a42ee1ec-d054-4a87-8344-0c7752cc3107.png)

Reply:
It does do that, doesn't it (see screenshot below)? The heading is Pool Candidates since the Pools heading is [something else](https://github.com/GCTC-NTGC/gc-digital-talent/pull/349), no?

![Screen Shot 2021-08-17 at 16 23 22](https://user-images.githubusercontent.com/3046459/129795510-0314eee9-c250-4d87-bb71-2c35c839e311.png)